src/app/store/in-memory-key/in-memory-key.selectors.ts

```diff
@@ -2,16 +2,23 @@ import { useSelector } from 'react-redux';
 
 import { createSelector } from '@reduxjs/toolkit';
 
+import { mnemonicToRootNode } from '@app/common/keychain/keychain';
+
 import { RootState } from '..';
 import { defaultKeyId } from '../keys/key.slice';
 
-export const selectInMemoryKey = (state: RootState) => state.inMemoryKeys;
+const selectInMemoryKey = (state: RootState) => state.inMemoryKeys;
 
 export const selectDefaultWalletKey = createSelector(
   selectInMemoryKey,
   state => state.keys[defaultKeyId]
 );
 
```

Comment:
`mnemonicToRootNode` seems to accept `secretKey: string`. 

Am I missing something that it needs `state` ?

Reply:
will rename